### PR TITLE
Updated pyaedt import to avoid error in IPY

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

**Comment by akaszynski** [comment]
This seems like a bad practice to me.  We should identify which module is causing the circular import and patch that (or change the order of imports) rather than importing everything twice.

**Comment by maxcapodi78** [comment]
@akaszynski it's a strange error. it only fails the first time you import a module, randomly when you run the script within AEDT. if you use Ironpython from outside AEDT it works fine.

**Comment by maxcapodi78** [comment]
@akaszynski in addition to this, I'm not able to reproduce it on my machine. Only happens randomly on some colleague machine

File: pyaedt/__init__.py [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
-from .icepak import Icepak
-from .edb import Edb
-from .maxwell import Maxwell2d, Maxwell3d
-from .mechanical import Mechanical
-from .rmxprt import Rmxprt
-from .simplorer import Simplorer
-from .desktop import Desktop
-from .emit import Emit
-
+try:
```
Comment (akaszynski) [inline pyaedt/__init__.py:2]:
```suggestion
# Import exception handling here due to:
# https://github.com/pyansys/PyAEDT/pull/243
try:
```

Comment (akaszynski) [reply pyaedt/__init__.py:2]: Added a note here so we know why there's this witchcraft here.

**Review by akaszynski — APPROVED** [review]
Added a comment so we could track the addition of the exception handling to this PR.

**Pull request merged**